src/app/sign_up/sign_up.scss

```diff
@@ -0,0 +1,56 @@
+input.ng-invalid.ng-touched { border: solid 1.5px #eb2a2a; }
```

Comment:
Keep in mind that here you're applying this style to all the inputs on the app. maybe you could add some classes and try to be more specific about its use